### PR TITLE
Add storybook

### LINKED ISSUE #2016: Add Storybook integration
Storybook is very helpful tool for creating, modifying and testing visual components. Also, Storybook integration requires that all visual components are independent from all HOCs like `react-router` etc.

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Resolves #2016

### CODE REVIEW

File: saleor/static/dashboard-next/components/CategoryList/CategoryList.tsx [hunk 1]
```diff
@@ -12,15 +12,20 @@ import i18n from "../../i18n";
...
 }
 export const CategoryList: React.StatelessComponent<CategoryListProps> = ({
   categories,
-  onClick
+  onClick,
+  loading
 }) => (
   <List>
-    {categories === undefined ? (
+    {loading ? (
```
Comment (patrys) [inline saleor/static/dashboard-next/components/CategoryList/CategoryList.tsx]: Why though? If we have a cache, we should do the initial render from cache.

File: saleor/static/dashboard-next/components/ProductList/ProductList.tsx [hunk 2]
```diff
@@ -67,7 +69,7 @@ export const ProductList = decorate<ProductListProps>(
         </TableRow>
       </TableFooter>
       <TableBody>
-        {products === undefined ? (
+        {loading ? (
```
Comment (patrys) [inline saleor/static/dashboard-next/components/ProductList/ProductList.tsx]: Same here.

File: saleor/static/dashboard-next/page/components/PageList/PageList.tsx [hunk 3]
```diff
@@ -23,13 +23,14 @@ interface PageListComponentProps {
   loading: boolean;
   handlePreviousPage();
   handleNextPage();
-  editPageUrl(id: string);
+  onEditClick(id: string);
   showPageUrl(slug: string);
```
Comment (patrys) [inline saleor/static/dashboard-next/page/components/PageList/PageList.tsx]: This should go. We should only pass events.

File: saleor/static/dashboard-next/components/CategoryList/CategoryList.tsx (hunk 1 above)
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/components/CategoryList/CategoryList.tsx]: I totally forgot about this!

**Comment by NyanKiyoshi** [comment]
I took a quick live look to the new dashboard. Just wanted to go by and say awesome job to all of you! Good luck! :tada:

**Pull request merged**